.gitignore

```diff
@@ -1,4 +1,7 @@
 # dependencies
 /node_modules
 /.next
-/.husky
\ No newline at end of file
+/.husky
+
+# local env files
+.env*.local
```

Comment:
hmmm this will be working for .env and .env-local? how?

Reply:
my env local file is env.local so it works fine